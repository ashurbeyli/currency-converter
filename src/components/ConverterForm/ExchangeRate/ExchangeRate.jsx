import React from 'react';

const ExchangeRate = ({ from, to }) => {
    return <span><strong>Exchange Rate</strong> 1{from} = 0.78{to}</span>;
};

export default ExchangeRate;