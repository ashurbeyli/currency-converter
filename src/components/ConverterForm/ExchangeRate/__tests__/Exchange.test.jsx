import React from 'react';
import { shallow } from 'enzyme';
import ExchangeRate from '../ExchangeRate';

describe('<ExchangeRate />', () => {
  test('snapshot should match', () => {
    const wrapper = shallow(<ExchangeRate from="EUR" currency="USD" />);
    expect(wrapper).toMatchSnapshot();
  });
});
