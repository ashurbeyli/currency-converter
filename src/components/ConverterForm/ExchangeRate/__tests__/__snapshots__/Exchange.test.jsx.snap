// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExchangeRate /> snapshot should match 1`] = `
<span>
  <strong>
    Exchange Rate
  </strong>
   1
  EUR
   = 0.78
</span>
`;
